modules/core/shared/src/main/scala/lucuma/core/data/Metadata.scala

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) 2016-2025 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.core.data
+
+import cats.collections.Diet
+import cats.collections.Range
+import cats.syntax.all.*
+import lucuma.core.enums.Instrument
+import lucuma.core.util.Enumerated
+
+import java.time.Instant 
+
+/** 
+ * A bundle of lookup tables and similar slow-changing but dynamic information
+ * from the ODB, intended to be loaded at application startup and passed 
+ * around implicitly. Rather than using this instance directly, methods on 
+ * data classes can provide better-placed methods that demand `Metadata`.
+ */
+trait Metadata:
```

Comment:
This will travel around, from ODB to UIs and servers. Can we add a `Codec` here? Or is a 2nd PR coming with the endpoint an the codec?

Reply:
Yeah the wire structure will be determined by the schema so I'll put this on the ODB side in a second PR